### PR TITLE
(2501) Show total of refunds in report summary

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Show total of refunds in report summary

### Before
Screenshot 2022-08-23 at 20 53 10

### After
Screenshot 2022-08-23 at 20 51 34

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
[Display total of refunds on report summary](https://trello.com/c/kUJcM5n8/2501-display-total-of-refunds-on-report-summary)

**Review by Gweaton — APPROVED** [review]
Nice, thanks for this. I'm approving but let's wait until the release has gone out for today before we merge this, as I think it's something BEIS will want to be aware of before we release it to users.

**Pull request merged**
